components/badge.json

```diff
@@ -4,6 +4,7 @@
     "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=4486%3A786",
```

Comment:
Could you help me updating this link to https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=46989%3A106273

Reply:
Component has been updated